src/front/app/components/challengeTopics/ChallengeTopic.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const ChallengeTopic = ({ challengeTopic = {} }) => (
+    <li>
+        <span>
+            {challengeTopic.name}
```

Comment:
this is bad practice, to pass whole challengeTopic object. component should know nothing about the model of chalange topic,
better to pass chalange topic name via props